easybuild/easyblocks/o/openfoam.py

```diff
@@ -292,6 +292,25 @@ def configure_step(self):
                         env.setvar("BOOST_ROOT", get_software_root('Boost'))
                     else:
                         env.setvar("%s_ROOT" % depend.upper(), dependloc)
+            for dep in self.cfg.dependencies():
+                # CGAL >= 5.x is header-only, but when using it OpenFOAM still needs MPFR and GMP.
+                # It may fail to find them, so inject the right settings and paths into the "have_cgal" script.
+                if dep['name'] == 'CGAL' and LooseVersion(dep['version']) >= LooseVersion('5.0'):
```

Comment:
Why not use `get_software_root('CGAL')` to check whether CGAL is loaded as dependency, and `get_software_version('CGAL')` to do the version check?

Reply:
Didn't know about `get_software_version`, that's way easier indeed!